libscpi/inc/scpi/types.h

```diff
@@ -114,10 +114,14 @@ extern "C" {
 
     typedef struct _scpi_command_t scpi_command_t;
 
-#if USE_COMMAND_TAGS
-	#define SCPI_CMD_LIST_END       {NULL, NULL, 0}
+#if USE_COMMAND_DESCRIPTIONS && USE_COMMAND_TAGS
```

Comment:
This does not scale. It would be nice to define some helper definitions to contain either nothing or `, NULL` / `, 0` and then construct the final `SCPI_CMD_LIST_END` unconditionally.